backend/src/main/java/com/glebabramov/backend/service/ResumeService.java

```diff
@@ -56,13 +57,57 @@ public Resume createResume(ResumeCreateRequest resume, Principal principal) {
 		return repository.save(newResume);
 	}
 
+	public Resume updateResume(ResumeRequest incomingResume, Principal principal) {
+		authorisationService.isAuthorisedByRole(ADMIN_ROLE, "update resumes", principal);
```

Comment:
I suggest renaming this method to something like `throwExceptionIfNotAuthorised` as the method doesn't even return true or false, but always true or throws an Exception :)